sky/core.py

```diff
@@ -102,6 +102,56 @@ def status(cluster_names: Optional[Union[str, Sequence[str]]] = None,
                                       cluster_names=cluster_names)
 
 
+@usage_lib.entrypoint
+def cost_report() -> List[Dict[str, Any]]:
```

Comment:
Add this function to `__init__.py` as well? Otherwise, we cannot access this function with `sky.cost_report`